test/browser_tests/spec_suites/careers-application-process.js

```diff
@@ -52,4 +52,31 @@ describe( 'The Application Process Page', function() {
     expect( page.relatedLinksSection.isPresent() ).toBe( true );
   } );
 
+  describe( '(Video Player)', function() {
+    it( 'should be present', function() {
+      expect( page.videoPlayer.isPresent() ).toBe( true );
+    } );
+
+    it( 'display show an image when the page loads', function() {
+      expect( page.videoPlayerImageContainer.isDisplayed() ).toBe( true );
+      expect( page.videoPlayerVideoContainer.isDisplayed() ).toBe( false );
+    } );
+
+    it( 'should show the video when the play button is clicked', function() {
+      page.videoPlayerPlayButton.click();
+      expect( page.videoPlayerVideoContainer.isDisplayed() ).toBe( true );
+      expect( page.videoPlayerImageContainer.isDisplayed() ).toBe( false );
+    } );
+
+    it( 'should attach an iframe when the play button is clicked', function() {
+      expect( page.getVideoPlayerIframe().isPresent() ).toBe( true );
+    } );
+
+    it( 'should hide the video when the close button is clicked', function() {
```

Comment:
Since you have two groups of interactions you might break it into two describe blocks:

```
describe Video Player
  describe "on page load"
    it …
    it …
  describe "on click"
    it …
    it …
```